File: app/src/main/java/com/quran/labs/androidquran/ui/adapter/DownloadedMenuActionListener.java
package com.quran.labs.androidquran.ui.adapter;

import com.quran.labs.androidquran.dao.translation.TranslationItem;

public interface DownloadedMenuActionListener {
  void startMenuAction(TranslationItem item, DownloadedItemActionListener downloadedItemActionListener);
  void finishMenuAction();
}

File: app/src/main/java/com/quran/labs/androidquran/ui/adapter/DownloadedMenuActionListener.kt
package com.quran.labs.androidquran.ui.adapter

import com.quran.labs.androidquran.dao.translation.TranslationItem

interface DownloadedMenuActionListener {
  fun startMenuAction(item: TranslationItem, downloadedItemActionListener: DownloadedItemActionListener?)
  fun finishMenuAction()
}
